tests/annotation_merge.nf.test

```diff
@@ -39,16 +40,17 @@ nextflow_pipeline {
         }
     }
 
-    test("Run with profile test | --tools vep --vep_include_fasta") {
+    test("Run with profile test | --tools merge,snpeff,vep") {
```

Comment:
looking at this we need to be more careful with the tool naming now that vcf merging (and not annotation merging) is coming cc @Patricie34